src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Github integration
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+
+<img
+  title="Github Integration dashboard"
+  alt="Github Integration dashboard"
+  src="/images/github-integration-screenshot.webp"
+/>
+
+<figcaption>
+After setting up our Github integration, see your data of teams, repos and users right out of the box.
+</figcaption>
+
+
+## Set up the Github integration [#set-up-github]
+
+Complete the following steps to set up the Github integration:
+
+<Steps>
+<Step>
+
+## Integration Details [#integration-details]
+
+By default, the Power Service catalog is selected, which will ensure to import all the data from Teams, Repositories and Users.
+    </Step>
+
+    <Step>
+
+## Github Integration [#github-integration]
+
+1. In this page, once we click on the install button, we get redirected to the Github App installation page. From there we need to install the Github app into the organization.
```

Comment:
Missing context: which page are we referring to here? Where are we performing these steps? Which GitHub app are we installing? Into which organization?

Reply:
Added context!